packages/teams-js/devdocs/logging.md

```diff
@@ -0,0 +1,43 @@
+# TeamsJS logging
+
+Enabling logging can be helpful when troubleshooting issues. These instructions are only for enabling local (browser console) logging in the TeamsJS library and will not enable logging within hosts like Teams, Outlook, or Office. However, even TeamsJS logging can be helpful to troubleshoot problems.
```

Comment:
> and will not enable logging within hosts like Teams, Outlook, or Office. 

This bit is a bit confusing to me. I think what it means is that Teams/Office/Outlook internally won't do any extra logging based on the enabling below. Is that the intent?

Reply:
I agree. Originally, when I was writing this, I was thinking about how there would be a different way to enable host logging -- but since I don't provide instructions on how to do that (and still have to figure out if we might want to ever do that) I'll just remove that for now.